advanced_functionality/tensorflow_iris_byom/tensorflow_BYOM_iris.ipynb

```diff
@@ -22,6 +22,13 @@
     "__Note__: Compare this with the [tensorflow bring your own model example](../tensorflow_iris_byom/tensorflow_BYOM_iris.ipynb)"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    IRIS_TRAINING_URL = "http://download.tensorflow.org/data/iris_training.csv"
</pre><p>While you're revising this notebook, can you please switch from the TensorFlow hosted version of Iris to our AWS-hosted version? S3 paths are:</p><pre class="ql-syntax" spellcheck="false">s3://sagemaker-sample-files/datasets/tabular/iris/iris_train.csv
s3://sagemaker-sample-files/datasets/tabular/iris/iris_test.csv
</pre><p>Then rather than using the <code>request</code>  library, you can use boto3.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Good idea!